### PR TITLE
Read data from registry on Windows (syscollector)

### PR DESCRIPTION
**Pull request by albertomn86**
Read Windows version, hostname and architecture directly from registry instead of using WMIC.

### CODE REVIEW

File: src/shared/version_op.c [hunk 1]
```diff
@@ -45,138 +52,102 @@ os_info *get_win_version()
...
-        command = "wmic os get Version";
-        cmd_output = popen(command, "r");
-        if (!cmd_output) {
-            merror("Unable to execute command: '%s'.", command);
-            info->os_version = strdup("unknown");
-        } else {
-            if (fgets(read_buff, buf_tam, cmd_output) && strncmp(read_buff, "Version", 7) == 0) {
-                if (!fgets(read_buff, buf_tam, cmd_output)){
-                    merror("Can't get version.");
-                    info->os_version = strdup("unknown");
```
Comment (vikman90) [inline src/shared/version_op.c]: Missing this attribute.

**Comment by vikman90** [comment]
I miss the column `os_version` in the Syscollector database:

Branch master:
```
> select * from sys_osinfo;
scan_id|scan_time|hostname|architecture|os_name|os_version|os_codename|os_major|os_minor|os_build|os_platform|sysname|release|version
1008794382|2018/04/20 00:42:58|WIN-DC13ROCR412|x86_64|Microsoft Windows Server 2016 Standard Evaluation|10.0.14393||10|0|14393||||
```

Branch fix-syscollector-win-osversion:
```
> select * from sys_osinfo;
scan_id|scan_time|hostname|architecture|os_name|os_version|os_codename|os_major|os_minor|os_build|os_platform|sysname|release|version
265247474|2018/04/20 00:29:05|WIN-DC13ROCR412|x86_64|Microsoft Windows Server 2016 Standard Evaluation|||10|0|14393||||
```

**Comment by chemamartinez** [comment]
Added: https://github.com/wazuh/wazuh/pull/547/commits/744eb707020164f3d255f66e1f9f73a82895fb91

```
sqlite> select * from sys_osinfo;
scan_id|scan_time|hostname|architecture|os_name|os_version|os_codename|os_major|os_minor|os_build|os_platform|sysname|release|version
1491148783|2018/04/20 17:38:03|WIN-NECNMD9IUBM|x86_64|Microsoft Windows Server 2016 Standard Evaluation|10.0.14393||10|0|14393||||6.2
```

**Comment by vikman90** [comment]
Thanks @chemamartinez !!

**Pull request merged**